components/Activity/QuestionnaireTable.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, Table } from 'react-bootstrap';
+
+import { Question } from '../../models/Question';
+import { i18n } from '../../models/Translation';
+
+const { t } = i18n;
+
+export interface QuestionnaireTableProps {
+  questionnaire: Question[];
+  onDelete: (id: string) => void;
+  onMove: (id: string, direction: number) => void;
+}
+
+export function QuestionnaireTable({
+  questionnaire,
+  onDelete,
+  onMove,
+}: QuestionnaireTableProps) {
+  const typeMap = {
+    text: t('text'),
+    url: t('link'),
+  };
```

Comment:
```suggestion
import { FC } from 'react';
import { observer } from 'mobx-react';

export const QuestionnaireTable: FC<QuestionnaireTableProps> = observer(({
  questionnaire,
  onDelete,
  onMove,
}) => {
  const typeMap = { text: t('text'), url: t('link') };
```

MobX i18n 需要 `observer()` 来更新